packages/firejail.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Firejail < Package
+  description 'Firejail is a SUID program that reduces the risk of security breaches by restricting the running environment of untrusted applications
+  by using Linux namespaces and seccomp-bpf.'
+  homepage 'https://firejail.wordpress.com'
+  version '0.9.54'
+  source_url 'https://sourceforge.net/projects/firejail/files/firejail/firejail-0.9.54.tar.xz'
```

Comment:
Please use `source_url 'https://prdownloads.sourceforge.net/project/firejail/firejail/firejail-0.9.54.tar.xz'` to prevent the download delay.